packages/lodestar/test/e2e/eth1/ethers.test.ts

```diff
@@ -0,0 +1,47 @@
+import "mocha";
+import {expect} from "chai";
+import {Eth1Provider, IEth1Block} from "../../../src/eth1";
+import {IEth1Options} from "../../../src/eth1/options";
+import {getMedallaConfig, medalla} from "./util";
+
+describe("eth1 / Eth1Provider", function () {
+  this.timeout("2 min");
+
+  const eth1Options: IEth1Options = {
+    enabled: true,
+    providerUrl: medalla.providerUrl,
```

Comment:
I don't think it's okay to run our e2e tests against prylabs public goerli node but
alternative is to setup private chain with deposit contract and deposits :disappointed:

Reply:
I think we were doing this before :sweat: 
Maybe we make an issue for this.

Reply:
Let's get some infura keys meanwhile?